modules/quality/include/opencv2/quality/QualityGMSD.hpp

```diff
@@ -0,0 +1,82 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#ifndef OPENCV_QUALITY_QUALITYGMSD_HPP
+#define OPENCV_QUALITY_QUALITYGMSD_HPP
+
+#include "QualityBase.hpp"
```

Comment:
Use lower case for file names: https://github.com/opencv/opencv/wiki/Coding_Style_Guide